src/components/ColorPicker/docs/ColorPickerFocusTabExample.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { classes } from './ColorPickerFocusTabExample.st.css';
+import { COLORS } from './ColorPickerExtendedExample';
+import { RadioButtonChangeEvent, RadioButtonClickEvent } from 'wix-ui-core/src/components/radio-button/RadioButton';
+import { ColorPicker } from '../ColorPicker';
+import classNames from 'classnames';
+
+interface State {
+  selectedColor: string;
+}
+
+interface Props {
+  className: string;
+}
+
+class ColorPickerStateful extends React.Component<Props, State> {
+  state = { selectedColor: null };
+
+  handleSelection = ({
+                       value,
+                     }: RadioButtonChangeEvent | RadioButtonClickEvent) => {
+    this.setState({ selectedColor: value });
+  };
+
+  render = () => (
+    <ColorPicker
+      {...this.props}
+      className={classNames(classes.colorPicker, this.props.className)}
+      onChange={this.handleSelection}
+    >
+      {COLORS.map((color, index) => (
+        <ColorPicker.Item
+          key={index}
+          value={color.value}
+          aria-label={color.ariaLabel}
+          checked={this.state.selectedColor === color.value}
+        />
+      ))}
+    </ColorPicker>
+  );
+}
+
+export class ColorPickerFocusTabExample extends React.Component {
+  render = () => (
+    <div className={classes.wrapper}>
+      <ColorPickerStateful className={classes.withFocusHighlight} />
+      <ColorPickerStateful className={classes.withFocusHighlight} />
+      <button className={classes.withFocusHighlight}>Select me</button>
+      <button className={classes.withFocusHighlight}>Select me 2</button>
```

Comment:
I'm not sure these buttons are necessary here

Reply:
This is the last block in the Settings section. After the last ColorPicker in this block the focus goes to browser's url input. It may be confusing when playing with focus behaviour so I added extra buttons.

Reply:
I think it's best to remove unrelated components in this tab, since it might cause some confusion. 
Shift-tab should still work, even when going passed the second ColorPIcker, 
and since there are 2 ColorPickers I think this example is pretty clear